lib/srv/desktop/rdp/rdpclient/src/client.rs

```diff
@@ -942,6 +942,7 @@ fn create_config(width: u16, height: u16, pin: String) -> Config {
         platform: MajorPlatformType::UNSPECIFIED,
         no_server_pointer: false,
         autologon: true,
+        pointer_software_rendering: false,
```

Comment:
Unrelated change?

Reply:
This (as well as changes in `web/packages/teleport/src/ironrdp/src/lib.rs`) was necessary to keep up with a changing IronRDP api. Ideally I would have done them as a separate commit, though it looks like I just lumped them all together.

I took a closer look at these settings and updated with 2e911c069f67c1aa7e07193db6809f3fb1b585bb